tools/keytools/keygen.c

```diff
@@ -294,12 +297,22 @@ int main(int argc, char** argv)
         }
     }
     output_pubkey_file = strdup(argv[argc - 1]);
+    dir = strdup(argv[argc - 1]);
```

Comment:
Looks like a command line argument has been added (`dir`) as positional, last argument? This should be at least documented in usage()

Reply:
Also ensure that keygen.c and keygen.py have the same interface and do the same thing (as @dgarske pointed out in his comment)

Reply:
I'll split that out as a separate PR. It's not necessary anymore for this PR.